src/panels/lovelace/components/hui-theme-select-editor.ts

```diff
@@ -57,6 +70,10 @@ export class HuiThemeSelectionEditor extends hassLocalizeLitMixin(LitElement) {
   }
 
   private _changed(ev): void {
+    if (!this.hass || this._initializing) {
```

Comment:
How is it possible that these things are fired before we got rendered? Is that on initialization?

Reply:
Yeah apparently...

Reply:
So is this PR solving problems or fighting symptoms? 😉 

In Polymer versions, we would always check if the value was `undefined` and ignore that one. I wonder if we can do something similar, because adding initialized logic to each place we use a dropdown won't scale.

Reply:
Also, instead of using `ev.target.value`, we can also inspect `ev.detail.value`, I wonder if it's different.